README.md

```diff
@@ -21,7 +21,49 @@ theme: minima
 And then execute:
 
     $ bundle
+
     
+## Contents At-A-Glance
+
+Minima has been scaffolded by the `jekyll new-theme` command and therefore has all the necessary files and directories to have a new Jekyll site up and running with zero-configuration.  
+
+### Layouts
+
+Refers to files within the `_layouts` directory, these files define the markup for your theme.  
+  
+  - `default.html` &mdash; The base layout that lays the foundation for subsequent layouts. The derived layouts inject their contents into this file at the line that says ` {{ content }} ` and are linked to this file via FrontMatter declaration  
```

Comment:
Maybe we can link to the FrontMatter documentation the first time we use the term: https://jekyllrb.com/docs/frontmatter/